changelog.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.29.1
```

Comment:
This shouldn't be a new entry. The version in version.txt is 0.29.0, so that's where your change entry should go.